src/workerd/api/cache.c++

```diff
@@ -518,7 +518,8 @@ jsg::Promise<bool> Cache::delete_(
 
 kj::Own<kj::HttpClient> Cache::getHttpClient(
     IoContext& context, kj::Maybe<kj::String> cfBlobJson, kj::ConstString operationName) {
```

Comment:
I think we should make `operationName` a LiteralConstString. Would this also simplify the next line?

Reply:
Thank you for the suggestion, that does make things easier! I tried to get the ConstString stuff right and the approach I had was memory safe at least but this is certainly more elegant/saves a memory allocation.